utils/ansible.py

```diff
@@ -0,0 +1,281 @@
+from distutils.sysconfig import get_python_lib
+import tempfile
+from os import listdir, mkdir, path
+from shutil import copy, copyfile, rmtree
+from subprocess import check_output, CalledProcessError, STDOUT
+
+from fauxfactory import gen_alphanumeric
+from utils import conf
+from utils.providers import get_crud
+
+from git import Repo
+from yaml import load, dump
+
+
+yml_path = path.join(path.dirname(__file__) + "/manageiq_ansible_module/")
```

Comment:
I was instructed before to put the manageiq ansible module library under utils where I have the ansible.py I need them all under the same library to work properly. I think it makes sense to have them there but I can put them in the root for the tests run and then to delete them afterwards.